src/modules/verifiableCredentials/verifiableCredentials.types.ts

```diff
@@ -48,3 +68,40 @@ export interface VerifyVerifiableCredentialResults {
   warnings: string[];
   errors: string[];
 }
+
+// Verifiable Presentation Types
+
+export interface SignVerifiablePresentationOptions {
+  verificationMethod?: string;
+  proofPurpose?: string;
+  domain?: Partial<TypedDataDomain>;
+}
+
+export interface UnsignedVerifiablePresentation {
+  '@context': string[];
+  id: string;
+  type: 'VerifiablePresentation';
+  holder: string;
+  verifiableCredential: SignedVerifiableCredential<unknown>[];
+}
+
+export interface VerifiablePresentation extends UnsignedVerifiablePresentation {
```

Comment:
Instead of `UnsignedVerifiablePresentation`, can just say `Presentation`. This is the term VC-API uses for example to refer to a VP that doesn't yet have a proof https://w3c-ccg.github.io/vc-api/holder.html#tag/presentation
```suggestion
export interface VerifiablePresentation extends Presentation {
```